CMakeLists.txt

```diff
@@ -493,6 +493,7 @@ set(WINDOW_FILES
     ${PROJECT_SOURCE_DIR}/src/window/config.c
     ${PROJECT_SOURCE_DIR}/src/window/difficulty_options.c
     ${PROJECT_SOURCE_DIR}/src/window/display_options.c
+	${PROJECT_SOURCE_DIR}/src/window/replay_confirm.c
```

Comment:
Keep files in alphabetical order, and keep formatting the same as the source (spaces vs tabs)